### ISSUE TITLE
Addons and CSS missing from the master build

### ISSUE THREAD

**Issue by Eugeny** [xtermjs/xterm.js#2178]
See above - it looks like the addons might be published each separately in the future?

**Comment by Tyriar** [comment]
Yes we pushed in https://github.com/xtermjs/xterm.js/pull/2164 but have not yet published the addon modules from that, you can probably still use the published xterm-addon-attach, xterm-addon-search and xterm-addon-web-links which were based on the now archived repos.

Let me know if that doesn't work and I'll prioritize https://github.com/xtermjs/xterm.js/issues/2165 a little higher.

**Comment by tsl0922** [comment]
Hi, @Tyriar 

The published `xterm-addon-web-links@0.1.0-beta1` doesn't work for me. 

Probably caused by the `main` field in `package.json`, it should be `lib/WebLinksAddon.js`.

```
{
  "name": "xterm-addon-web-links",
  "version": "0.1.0-beta1",
  "main": "webLinksAddon.ts",
  "scripts": {
    "watch": "tsc -w"
  },
  "devDependencies": {
    "typescript": "^3.2.2",
    "xterm": "^3.10.0"
  },
  "peerDependencies": {
    "xterm": "^3.10.0"
  }
}

```

**Comment by Tyriar** [comment]
@tsl0922 I accidentally published that under the latest tag, `npm i xterm-addon-web-links@beta` should work
